lib/tasks/rubocop-ci.rake

```diff
@@ -95,6 +95,11 @@ if Dir.exist?('app')
     end
   end
 
+  task :rubocop do
+    files = Dir['config/locales/**/*.yml'].join(' ')
+    sh "i18n-lint #{files}"
```

Comment:
If I'm not mistaken this might blow up with "Argument list too long" in case there are a lot of yml files or long path names involved.

Reply:
It works for `standard` with .jsx files, and I expect there will be much less .yml files, so should be fine.